app/services/siade.rb

```diff
@@ -0,0 +1,56 @@
+class Siade
+  attr_reader :token
+
+  def initialize(token:)
+    @token = token
+  end
+
+  def entreprises(siret:)
+    endpoint = "v2/entreprises/#{siret}"
+
+    siade_request(endpoint)
+  end
```

Comment:
```suggestion
  def entreprises(siret:)
    siade_request("v2/entreprises/#{siret}")
  end
```

cette affectation de variable est overkill imo (valide pour les 3 du dessous).